grammars/gfm.cson

```diff
@@ -145,7 +145,7 @@
   }
   {
     'begin': '\\A---$'
-    'end': '^---$'
+    'end': '^---|\\.\\.\\.$' 
```

Comment:
Done. Although it worked without adding a group according to RegexPlanet's Ruby processor. Thanks for the followup.